sql/parser/sql.y

```diff
@@ -572,7 +602,10 @@ alter_table_cmd:
   //     [ USING <expression> ]
 | ALTER opt_column name opt_set_data TYPE typename opt_collate_clause alter_using {}
   // ALTER TABLE <name> ADD CONSTRAINT ...
-| ADD table_constraint {}
+| ADD table_constraint
```

Comment:
Probably not. I was just talking out loud. There is a lot of alternate syntax in the grammar. This is not the only oddity.